crates/rolldown/src/module_loader/runtime_normal_module_task.rs

```diff
@@ -80,6 +80,9 @@ impl RuntimeNormalModuleTask {
       import_records: IndexVec::default(),
       is_included: false,
       sourcemap_chain: vec![],
+      // The internal runtime module `importers/dynamic_importers` should be skip.
+      importers: vec![],
```

Comment:
Would it be better if we collect these datas in `module_loader` instead of worker? Since we are running in parallel, these's big chance the those values is not stable cross builds.

Reply:
Move it to `module_loader` couldn't make it stable, i add `sort` at getter.